### PR TITLE
Use unifiedAppDeployment flag instead of appUiDeployments

### PR DESCRIPTION
**Pull request by matteodepalo**
### WHY are these changes introduced?

We want to move the beta flag for unified deployments to the org level because we're getting rid of the org level one and also because the partners token doesn't have permission to fetch organizations, so the `deploy` command couldn't make use of such flags.

### WHAT is this pull request doing?

I've removed `appUiDeployments` from organizations and added `unifiedAppDeployment` to apps.

### How to test your changes?

- Enable the `unified_app_deployment` beta flag in partners
- The new flow should be active when deploying

### CODE REVIEW

File: packages/app/src/cli/api/graphql/all_orgs.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export interface AllOrganizationsQuerySchemaOrganization {
   businessName: string
   website?: string
   betas: {
-    appUiDeployments?: boolean
+    cliTunnelAlternative?: boolean
```
Comment (Arkham) [inline packages/app/src/cli/api/graphql/all_orgs.ts:8]: ??

Comment (matteodepalo) [reply packages/app/src/cli/api/graphql/all_orgs.ts:8]: @isaacroldan I've seen this flag pop up recently but I haven't seen it queried for consistently so this change brings consistency.

File: packages/app/src/cli/services/deploy.test.ts [hunk 2]
```diff
@@ -234,8 +284,8 @@ async function testDeployBundle(app: AppInterface, organization?: Organization,
   await deploy({
     app,
     reset: false,
-    force: true,
-    label,
+    force: options?.force ?? false,
```
Comment (Arkham) [inline packages/app/src/cli/services/deploy.test.ts:287]:
```suggestion
    force: Boolean(options?.force),
```

is this the same?

File: packages/app/src/cli/services/context.ts [hunk 3]
```diff
@@ -317,24 +315,21 @@ export async function ensureDeployContext(options: DeployContextOptions): Promis
...
 
   await logMetadataForLoadedDeployContext(result)
   return result
 }
 
-export async function fetchOrganizationAndFetchOrCreateApp(
-  app: AppInterface,
-  token: string,
-): Promise<{partnersApp: OrganizationApp; organization: Organization}> {
+export async function fetchOrganizationAndFetchOrCreateApp(app: AppInterface, token: string): Promise<OrganizationApp> {
```
[the remark this answers is not in the document]
Comment (matteodepalo) [reply packages/app/src/cli/services/context.ts:328]: Good point 👍

**Pull request merged**